ConsentViewController/Classes/SourcePointClient/Adapters.swift

```diff
@@ -501,6 +556,7 @@ extension SourcepointClientCoordinator.State {
             ccpa: self.ccpa.toCore(metaData: self.ccpaMetaData),
             usNat: self.usnat.toCore(metaData: self.usNatMetaData),
             ios14: self.ios14.toCore(),
+            preferences: emptyPreferencesCampaign(),
```

Comment:
Why are we initializing state with empty preferences data?

Reply:
@andresilveirah this is used as a part of data migration from native SDK data storage to mobile core data storage on the first-ever launch. 
There are 2 use cases:
1. The user had some other campaign data in the native SDK storage. Since we previously didn't have `preferences`, we need to fill the state with an empty `preferences` campaign object.
2. If there is no consent exists in data storage, neither in native nor in core, we need an empty `preferences` to initialize the core's data storage